x/poolmanager/router_test.go

```diff
@@ -1566,6 +1566,7 @@ func (s *KeeperTestSuite) TestSingleSwapExactAmountIn() {
 			if (tc.takerFee != osmomath.Dec{}) {
 				// If applicable, set taker fee. Note that denoms are reordered lexicographically before being stored.
 				poolmanagerKeeper.SetDenomPairTakerFee(s.Ctx, tc.poolCoins[0].Denom, tc.poolCoins[1].Denom, tc.takerFee)
+				poolmanagerKeeper.SetDenomPairTakerFee(s.Ctx, tc.poolCoins[1].Denom, tc.poolCoins[0].Denom, tc.takerFee)
```

Comment:
We now set for both direction as two separate calls. The upgrade handler transforms all existing denom pair taker fees to automatically go in both directions.